rohmu/object_storage/local.py

```diff
@@ -207,18 +213,23 @@ def store_file_object(
         target_path = self.format_key_for_backend(key.strip("/"))
         os.makedirs(os.path.dirname(target_path), exist_ok=True)
         bytes_written = 0
+        m = hashlib.sha256()
```

Comment:
nit: this is a bit weird, but from what I see it's already done for, e.g., azure etags and other stuff so ok for me.

I see that SFTP also lacks the hash field.